src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -87,7 +84,22 @@ Copyright (c) .NET Foundation. All rights reserved.
     <SupportedPlatform Include="browser" />
   </ItemGroup>
 
-  <Import Project="Microsoft.NET.Sdk.BlazorWebAssembly.ServiceWorkerAssetsManifest.targets" Condition="'$(ServiceWorkerAssetsManifest)' != ''" />
```

Comment:
As spoken yesterday, I didn't have that at the time, I've broken down things by version now so that 5.0 and earlier use the old version, and 6.0 and onwards use the new version